mbuild/tests/test_compound.py

```diff
@@ -1164,11 +1165,25 @@ def test_from_pybel_molecule(self, extension):
         #            chol.unitcell.GetC()/10],
         #        rtol=1e-3)
 
+    @pytest.mark.parametrize('test_smiles', [
+        "CCO",
+        "CCCCCCCC",
+        "c1ccccc1,",
```

Comment:
```suggestion
        "c1ccccc1",
```
oops, extra comma still causing tests to fail